notebooks/Run generative finetuning.ipynb

```diff
@@ -154,7 +150,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Note:** Repeat for the same for the evaluation dataset if you have one. If absent, we will auto-split the training dataset into training and evaluation datasets."
+    "**Note:** Repeat for the same for the evaluation dataset if you have one. If absent, we will auto-split the training dataset into training and evaluation datasets with the ratio of 90:10.\n",
+    "\n",
+    "Remember the datset must contain at least 32 records This ratio is overwritten if the evaluation dataset is less 16 records. So for small datasets the evaluation is at least 16 records. We recommend using a dataset than is 100 or more examples large. A larger dataset is likely to yield high quality finetunes. Be aware that a larger dataset means time to finetune will also be larger."
```

Comment:
nit: wording is a bit confusing, maybe something like:
If no evaluation dataset is provided, the dataset must contain at least 32 records. If an evaluation dataset is provided, both training and evaluation datasets must be at least 16 records. We recommend using a dataset that is at least 100 examples. A larger dataset is likely to yield high quality finetunes, but be aware that a larger dataset means time to finetune will also be larger.

Reply:
Reworded to 


Remember the dataset must contain at least 32 examples. If an evaluation dataset is provided, both training and evaluation datasets must contain at least 16 examples. The above split ratio is overwritten if the evaluation split is lesser than 16 examples. So for a dataset of size 50 the evaluation is 16 examples and the remaining 34 examples are used for training.

We recommend using a dataset than contains at least 100 examples but a larger dataset is likely to yield high quality finetunes. Be aware that a larger dataset would mean that the time to finetune would also be larger.

WDYT?